packages/bruno-app/src/utils/exporters/postman-collection.js

```diff
@@ -173,14 +173,19 @@ export const exportCollection = (collection) => {
 
   const generateHost = (url) => {
     try {
-      const { hostname } = new URL(url);
-      return hostname.split('.');
+      if (url.includes('{{') && url.includes('}}')) {
+        return url;
```

Comment:
const { hostname } = new URL(url); has the effect of removing the path after the hostname. It will not work with that code. Also, we should add unit tests :)